lib/iris/tests/unit/analysis/test_Aggregator.py

```diff
@@ -315,6 +319,32 @@ def test_kwarg_pass_through_combined_kwargs(self):
         expected_kwargs.update(call_kwargs)
         lazy_func.assert_called_once_with(data, axis, **expected_kwargs)
 
+class Test_aggregated_by(tests.IrisTest):
+    def setUp(self):
+        latitude = DimCoord(np.linspace(-90, 90, 4),
+                            standard_name='latitude',
+                            units='degrees')
+        longitude = DimCoord(np.linspace(45, 360, 8),
+                             standard_name='longitude',
+                             units='degrees')
+        time_unit = unit.Unit('hours since epoch',
+                              calendar=unit.CALENDAR_360_DAY)
+        time = DimCoord(np.linspace(104040, 122040, 26),
```

Comment:
Yeah sure.  I have updated the description (although I can add more detail if necessary, but it will get long-winded I expect) and tried to simplify the test.  Do you want it simplified more?